.github/workflows/project-ticket-observer.yml

```
@@ -0,0 +1,285 @@
+# ============================================================
+on:
+  # 1. When a new issue is created, edited, assigned, labeled, etc.
+  issues:
+    types: [opened, edited, assigned, unassigned, labeled, unlabeled, closed, reopened]
+
+  # 2. When project cards change (classic projects)
+  project_card:
+    types: [created, moved, converted, edited, deleted]
+
+  # 3. When project v2 items change (new GitHub Projects)
+  projects_v2_item:
```

actionlint (1.7.11)

[error] unknown Webhook event "projects_v2_item". see https://docs.github.com/en/actions/reference/workflows-and-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)